src/app/dock_widgets/process/gt_processdock.cpp

```diff
@@ -1863,7 +1863,7 @@ GtProcessDock::pasteElement(GtObject* obj, GtObject* parent)
 
     m_view->setFocus();
 
-    auto pasteCmd = gtApp->makeCommand(gtApp->currentProject(),
+    auto pasteCmd = gtApp->makeCommand(m_taskGroup,
```

Comment:
You may be able to use parent here
```suggestion
    auto pasteCmd = gtApp->makeCommand(parent,
```